index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+    
+</body>
```

Comment:
- [ ] [Optional] How about wrapping the h1 element in a header tag, to more semantic meaning to the project? 🤔

Reply:
Definitely, I have to put more attention to what am I doing  😅   😅
Thanks again, Daniel!

Reply:
> Definitely, I have to put more attention to what am I doing 😅 😅
> Thanks again, Daniel!

Nice